.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Bugs in third-party projects (e.g. `requests`) do not belong in the CPython issue tracker
+      options:
+        - label: I am confident this is a bug in CPython, not a bug in a third-party project
+          required: true
+        - label: |
+            I have searched the [CPython issue tracker](https://github.com/python/cpython/issues),
+            and am confident this bug has not been reported before
+          required: true
+  - type: textarea
+    attributes:
+      label: A clear and concise description of the bug
+      description: |
+        Tell us what happened.
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Put any code blocks inside triple backticks.
+
+      value: |
+        ```py
+        your code here
+        ```
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: CPython versions tested on
+      multiple: true
+      options:
+        - "3.8"
+        - "3.9"
+        - "3.10"
+        - "3.11"
+        - "3.12"
+        - "CPython main branch"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Operating systems and architectures tested on
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Any other relevant information
+    validations:
+      required: false
```

Comment:
Personal opinion, but I'm not a fan of "other" fields here -- people tend to use the first free-response field to describe everything relevant (to them), and so these "other" fields often are empty and a distraction to readers of the issue.

Reply:
If this is removed, I would move the version and os info before the main "bug description" textarea, like it was on bpo.  This way people will fill all the values and then write everything else at the bottom, in the only available textarea.

IOW: disclaimer about looking for help, checklist, version dropdown, OS dropdown, description textarea.

Reply:
I'd agree with this

Reply:
Applied!